### ISSUE TITLE
Several old movies marked as In Cinemas

### ISSUE THREAD

**Issue by jpborjas** [Radarr/Radarr#1038]
After the latest release 0.2.0.453, now several old movies are being marked as In Cinemas. 

![screen shot 2017-03-06 at 9 23 39 am](https://cloud.githubusercontent.com/assets/14253583/23614049/a82bb7a8-024f-11e7-8781-8617f403bae9.png)

If I do a scan, or manually look for the movies folder, do a manual update, and then go back to the main page, I'll see all those movies there, even though I'm filtering my view to Missing only. If I select In Cinemas, and then back to Missing Only, they go away. 

Steps to reproduce:

While on the main Page, I see movies (and concerts) marked as In Cinemas, even though I'm selecting Missing Only.

![screen shot 2017-03-06 at 9 34 18 am](https://cloud.githubusercontent.com/assets/14253583/23614258/4ff41598-0250-11e7-8969-b3e4ee02afbe.png)

If click on the In Cinemas Filter button, and then back on Missing Only, then I get the right results:

![screen shot 2017-03-06 at 9 40 07 am](https://cloud.githubusercontent.com/assets/14253583/23614516/2ff37d00-0251-11e7-93fa-fc4454ee68ce.png)

![screen shot 2017-03-06 at 9 44 13 am](https://cloud.githubusercontent.com/assets/14253583/23614604/800eedc4-0251-11e7-99e4-bb6fd7f0fe91.png)

**Radarr Version: 0.2.0.453 **

**Logs:**

[radarr.txt](https://github.com/Radarr/Radarr/files/821478/radarr.txt)

Please use the search bar and make sure you are not submitting an already submitted issue.
Visit our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by geogolem** [comment]
there was a problem regarding filters when returning to the main movies page. The toolbar button would show the previously used fitler, but all the movies were being shown. I have put a fix in for this.. Im not sure if this is exactly what you were experiencing but could you please try it with the latest and let me know.
Thanks

**Comment by geogolem** [comment]
It is possible you will see movies "In Cinema" when "Missing Only".. because they are not exclusive.

a movie can be in cinemas and missing, as a matter of fact it is likely that a movie tht is in Cinemas will be missing...

Missing means all movies for which no movie download exists....

**Comment by jpborjas** [comment]
This was experienced with the version released today, and the movies that I'm referring to, are all OLD movies. They are definitely not In Cinemas.

**Comment by galli-leo** [comment]
@jpborjas Did you do a library update?

**Comment by jpborjas** [comment]
All the tasks have already ran at least once today.

**Comment by geogolem** [comment]
I put in a few fixes just before i made my last comment.. around 3 hours ago.. so,  unless you are running the latest, even if you have the pre-release that was released last night or this morning, you likely still dont have the fixes.

**Comment by mgiljum** [comment]
I can confirm the issue for v0.2.0.453.

Fresh install and an import of a few hundred different moves and they are all being marked as "In Cinemas", and it seems all movies added after that point show as In Cinemas as well.

**Comment by mgiljum** [comment]
Possibly related to #511 ?

**Comment by geogolem** [comment]
I dont seem to be having this problem at all.

This is how radarr works internally with respect to deciding the movie status:
First it grabs the release date info from TMDB.
then based on the release date information available it makes a decision if the movie is announced, inCinemas or Released.

then since many old movies would be marked as inCinemas simply because TMDB lacks release date info. We mark any movie that is InCinemas that doesnt have physical release date info as physically released if it was in Cinemas > 3 months ago.

I just looked at the code, and it seems to make sense.. and I also did a full library refresh with a clean develop build and I didnt get anything unexpected or unusual.

------------------

To clarify: A movie being "InCinemas" or "Not Monitored" doesnt mean its not "Missing"  Referring to the original post --> there was a bug with respect to filtering, where when returning to the movie Index after visiting another page (i.e. settings) the filter toolbar would still show your previously used filter; however all movies would be displayed.... this was fixed..

The only reason I am keeping this issue open and scratching my head is because of your first image shows movies like "Monsters Inc" tagged as InCinemas.. this shouldnt happen...  But in my testing, this is correctly tagged as Released...  Could you please get back to me with more information.

Also keep in mind that the release dates and release status of movies is only updated during a library refresh or when the individual movie itself is refreshed (and disk is scanned)

**Comment by mgiljum** [comment]
@geogolem After restarting Radarr this morning, the problem is gone on my clone, with existing monitored movies and newly added ones. Strange, must be something to do with cache.

**Comment by geogolem** [comment]
ok - i am going to close this issue off; however if you see this issue again please let me know. Thanks

**Comment by jpborjas** [comment]
This is still an issue, I've restarted to no avail.

**Comment by geogolem** [comment]
@jpborjas : Can you please provide more information and confirm exactly what the issue is. Are you running the latest nightly driver? Have you refreshed all the movieInfo in your library.

**Comment by jpborjas** [comment]
I'm not running nightly. I trie dto install it once and didn't work for my Mac. Let me try again.

**Comment by jpborjas** [comment]
I couldn't get the nightly version. I'm running 0.2.0.453 for Mac. Where do I get the nightly?

**Comment by jpborjas** [comment]
I didn't know how to switch to nightly. I did it and now it works. Thanks!
